include/global.h

```diff
@@ -928,16 +928,16 @@ struct SaveBlock1
     /*0x159C*/ u32 gameStats[NUM_GAME_STATS];
     /*0x169C*/ struct BerryTree berryTrees[BERRY_TREES_COUNT];
     /*0x1A9C*/ struct SecretBase secretBases[SECRET_BASES_COUNT];
-    /*0x271C*/ u8 playerRoomDecor[DECOR_MAX_PLAYERS_HOUSE];
-    /*0x2728*/ u8 playerRoomDecorPos[DECOR_MAX_PLAYERS_HOUSE];
-    /*0x2734*/ u8 decorDesk[10];
-    /*0x273E*/ u8 decorChair[10];
-    /*0x2748*/ u8 decorPlant[10];
-    /*0x2752*/ u8 decorOrnament[30];
-    /*0x2770*/ u8 decorMat[30];
-    /*0x278E*/ u8 decorPoster[10];
-    /*0x2798*/ u8 decorDoll[40];
-    /*0x27C0*/ u8 decorCushion[10];
+    /*0x271C*/ u8 playerRoomDecorations[DECOR_MAX_PLAYERS_HOUSE];
+    /*0x2728*/ u8 playerRoomDecorationPositions[DECOR_MAX_PLAYERS_HOUSE];
+    /*0x2734*/ u8 decorationDesks[10];
+    /*0x273E*/ u8 decorationChairs[10];
+    /*0x2748*/ u8 decorationPlants[10];
+    /*0x2752*/ u8 decorationOrnaments[30];
+    /*0x2770*/ u8 decorationMats[30];
+    /*0x278E*/ u8 decorationPosters[10];
+    /*0x2798*/ u8 decorationDolls[40];
+    /*0x27C0*/ u8 decorationCushions[10];
```

Comment:
Any particular reason for the increased verbosity?

Reply:
The `SecretBase` struct uses "decoration", and I assumed that the whole word would be preferred to the abbreviated form. Happy to switch everything to "decor" if you prefer. Or could leave them mixed, but I think that comes across a bit weird?

Reply:
For what it's worth, I have never liked the `decor` abbreviation and would prefer `decoration`, in general.